Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "AdvancedHMC"
 uuid = "0bf59076-c3b1-5ca4-86bd-e02cd72cde3d"
-version = "0.2.20"
+version = "0.2.21"
```

Comment:
@JuliaRegistrator register

Reply:
I think you'd have to do it as a commit comment. It would be great if you could use this one https://github.com/TuringLang/AdvancedHMC.jl/commit/20d5350fc6bc60fde6c5cb31243d222ab11bacb7 as it would also include the LazyArrays change